beerlist with B/Model/RandomBeerAPI.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+struct Beer: Codable, Identifiable {
+    var name: String
+    var imageURL: String
+    var description: String
+    var beerId: Int
+    let id = UUID()
+    
+    enum CodingKeys : String, CodingKey {
+        case name
+        case imageURL = "image_url"
+        case description
+        case beerId = "id"
+    }
+}
+
+final class RandomBeerAPI {
+    
+    func getRandomBeerData(completion: @escaping (Beer?) -> ()) {
+        
+        guard let url = URL(string: "https://api.punkapi.com/v2/beers/random") else {
+            return
+        }
+        
+        URLSession.shared.dataTask(with: url) {(data, response, error) in
+            guard let data = data else {return}
```

Comment:
```suggestion
            guard let data else { return }
```
Swift 5.8부터 if, guard let/var 에서 이름 똑같이 쓰면 = 을 생략 가능합니다

Reply:
아 5.7이네

Reply:
if let shorthand 라구 검새가면 관련 내용 나올거에요